### ISSUE TITLE
incomplete data annotation

### ISSUE THREAD

**Issue by lazyleaf** [fchollet/keras#5911]
Dear Community,

I am working with a fully convolutional NN (no fully connected layers) to perform segmentation of images. So the output of my network is not a single label, but an array of labels. Therefore, for training I feed to my network a series of data array - label array pairs. The thing is, not all my data has been annotated: there are elements of my label array targets which are unknown. How do I tell keras to ignore those unknown labels? For example, in elektronn (another deep learning framework in python) I just have to put the value '-1' for an unknown label, so that this particular pixel is ignored during training. Is there something similar in keras?

Best,
Lazyleaf

**Comment by hko-gh** [comment]
I am having the exact same question. Could we solve it by using a custom loss function that ignores such pixels?
